charts/neon-proxy/templates/deployment.yaml

```diff
@@ -187,7 +187,9 @@ spec:
             {{- with .Values.settings.awsRegion }}
             - --aws-region={{ . }}
             {{ end }}
-
+            {{- with .Values.settings.extraCmdFlags }}
+            {{- toYaml . | nindent 2 }}
```

Comment:
should the indent be 2, 12, or 14? I never know with helm

Reply:
it should be 12

Reply:
yes, its 12. thanks @conradludgate